src/RealWorld/Conduit/Users/Web/Current/Update.hs

```diff
@@ -36,9 +36,17 @@ data UserUpdate = UserUpdate
   }
 
 deriving instance Generic UserUpdate
-deriving instance FromJSON UserUpdate
 deriving instance ToSchema UserUpdate
 
+instance FromJSON UserUpdate where
```

Comment:
Yeah, this is the only way that I know how to do it with that type, too. 

The other way would be to send a DMap instead of a product type, but I dunno how that'd work with servant.

Reply:
This exists: http://hackage.haskell.org/package/dependent-sum-aeson-orphans-0.2.0.0/docs/Data-Dependent-Sum-Orphans.html